tests/integration/tokenomics/token_logic_modules/commutative_test.go

```diff
@@ -0,0 +1,197 @@
+// from SetupTest(). It also sets the block height to 1 and the proposer address to
+// the proposer address from SetupTest().
+func (s *tlmProcessorTestSuite) setupKeepers(t *testing.T, opts ...keeper.TokenomicsModuleKeepersOptFn) {
+	defaultOpts := []keeper.TokenomicsModuleKeepersOptFn{
+		keeper.WithService(*s.service),
+		keeper.WithApplication(*s.app),
+		keeper.WithSupplier(*s.supplier),
+		keeper.WithModuleParams(map[string]types.Msg{
+			// TODO_TECHDEBT: Set tokenomics mint allocation params to maximize coverage, once available.
+
+			// Set the proof params such that proofs are NEVER required.
+			prooftypes.ModuleName: s.getProofParams(),
+			// Set the CUTTM to simplify calculating settlement amount expectstions.
+			sharedtypes.ModuleName: s.getSharedParams(),
+		}),
+	}
+
+	s.keepers, s.ctx = keeper.NewTokenomicsModuleKeepers(
+		t, log.NewNopLogger(),
+		append(defaultOpts, opts...)...,
+	)
+
+	// Increment the block height to 1; valid session height and set the proposer address.
+	s.ctx = types.UnwrapSDKContext(s.ctx).
+		WithBlockHeight(1).
+		WithProposer(s.proposerConsAddr)
+}
+
+// setExpectedSettlementState sets the expected settlement state on the suite based
+// on the current network state and the given settledResults and expiredResults.
+func (s *tlmProcessorTestSuite) setExpectedSettlementState(
+	t *testing.T,
+	settledResults,
+	expiredResults tlm.PendingSettlementResults,
+) {
+	t.Helper()
+
+	s.expectedSettledResults = settledResults
+	s.expectedExpiredResults = expiredResults
+	s.expectedSettlementState = s.getSettlementState(t)
+}
+
+// getSettlementState returns a settlement state based on the current network state.
+func (s *tlmProcessorTestSuite) getSettlementState(t *testing.T) *settlementState {
+	t.Helper()
+
+	app, isAppFound := s.keepers.GetApplication(s.ctx, s.app.GetAddress())
+	require.True(t, isAppFound)
+
+	proposerBech32 := sample.AccAddressFromConsBech32(s.proposerConsAddr.String())
+
+	return &settlementState{
+		appStake:             app.GetStake(),
+		supplierOwnerBalance: s.getBalance(t, s.supplier.GetOwnerAddress()),
+		proposerBalance:      s.getBalance(t, proposerBech32),
+		foundationBalance:    s.getBalance(t, s.foundationBech32),
+		sourceOwnerBalance:   s.getBalance(t, s.sourceOwnerBech32),
+	}
+}
+
+// getBalance returns the current balance of the given bech32 address.
+func (s *tlmProcessorTestSuite) getBalance(t *testing.T, bech32 string) *types.Coin {
+	t.Helper()
+
+	res, err := s.keepers.Balance(s.ctx, &banktypes.QueryBalanceRequest{
+		Address: bech32,
+		Denom:   volatile.DenomuPOKT,
+	})
+	require.NoError(t, err)
+	require.NotNil(t, res)
+
+	return res.GetBalance()
+}
+
+// assertExpectedSettlementState asserts that the current network state matches the
+// expected settlement state, and that actualSettledResults and actualExpiredResults
+// match their corresponding expectations.
+func (s *tlmProcessorTestSuite) assertExpectedSettlementState(
```

Comment:
Is there any way to update the test such that we moe forward one (or two) more sessions and confirm the state/balances are unchanged (since there are no new claims).

This is likely a different test, but just want to sanity checks the session start/end edge case + lingering claims.